index.html

```diff
@@ -390,11 +390,23 @@ <h2>Coordination</h2>
           <a href="https://www.w3.org/policies/process/#RecsCR" title="Candidate Recommendation">CR</a> and is encouraged
           to proactively notify the horizontal review groups when major changes occur in a specification following a review.</p>
 
-        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/policies/process/#WGCharter">W3C Process Document</a>:</p>
+        <section>
+          <h3 id="relationship-with-ddi">Relationship with the DDI Alliance</h3>
+
+          <p>The DDI Alliance and the W3C establish a Working Group within the W3C aimed at producing a Recommendation and other deliverables as described above, which will include significant contributions from both the W3C and DDI (Data Documentation Initiative) communities. The cooperation is done on the basis of a liaison and a formal agreement. The planned deliverables will be endorsed by both organizations. The Working Group will operate under the W3C Patent Policy, and will use the W3C Software and Document license for all its deliverables. Membership in the group will include representatives of W3C member organizations, representatives of DDI Alliance member organizations, and invited experts.
+</p>
+
+          <p>As is now common practice for both organizations, the W3C group will work in public, i.e. it will use a publicly visible mailing list and wiki, and the editor's drafts of its documents will be publicly visible (e.g. on GitHub). To enable write access to these facilities, unless there are specific reasons to the contrary, members of the Working Group who do not represent W3C member organizations will be granted Invited Expert status, with all obligations and privileges, in the WG but without access to member-only resources.
```

Comment:
```suggestion
          <p>As is now common practice for both organizations, the W3C Working Group will work in public, i.e. it will use a publicly visible mailing list and wiki, and the editor's drafts of its documents will be publicly visible (e.g. on GitHub). To enable write access to these facilities, unless there are specific reasons to the contrary, participants of the Working Group who do not represent W3C member organizations will be granted Invited Expert status, with all obligations and privileges, in the Working Group but without access to member-only resources.
```